.gitignore

```diff
@@ -3,3 +3,9 @@
 
 # Ignore local virtual machine configuration
 config.yaml
+
+# Ignore vagrant's metadata dir
+.vagrant
+
+# Ignore ide's metadata dir
+.idea
```

Comment:
Could you remove this line? It's an ignore rule for a specific editor configuration, not really related to this project.

Reply:
it's removed.